### ISSUE TITLE
pnpm install <package> does not trigger postinstall script

### ISSUE THREAD

**Issue by tomitrescak** [pnpm/pnpm#1489]
### Code to reproduce the issue:

pnpm install "package"

### Expected behavior:

The "postinstall" script from package.json is executed

### Actual behavior:

Does not execute

### Additional information:

postinstall only executes when doing `pnpm install` without package name

 - `node -v` prints: v10.9.0
 - Windows, OS X, or Linux?: OS X

**Comment by zkochan** [comment]
That's how it works with npm as well.

**Comment by tomitrescak** [comment]
@zkochan fair enough! I wrote a blog post about pnpm several years ago and it's my most visited post on Medium ;) I haven't been using it for many years though, just got back to ity and found out it came a long way. What everyone else makes so complex this makes it easy. I just wish I did not have to write --shamfully-flatten all the time. Unfortunately packages I work with attract a lot of unresolved peer dependencies. Thanks again.
